README.md

```diff
@@ -30,3 +30,11 @@ npm run watch #builds client package
 npm run build # build
 # Checkout package.json for more tasks
 ```
+
+### Logs :
+
+Below list presents specific log messages for Segment Connector :
+
+* incoming.screen.success - logged on successfully screen handling
+* incoming.track.success - logged on successful track handling
+* outgoing.group.success - batch of incoming users updated separately. It will also cause to update metric for each updated user
```

Comment:
Yes, but they are logged at error level - should I document error level logs within README.md ?
I assumed that al info level logs should be documented in README, because they are treated differently. 
Maybe this incoming.*.error logs that you mentioned should be in info level and then we should put them into README ?